io/src/obj_io.cpp

```diff
@@ -592,6 +592,10 @@ pcl::OBJReader::read (const std::string &file_name, pcl::PCLPointCloud2 &cloud,
       // Vertex normal
       if (st[0] == "vn")
       {
+        if (normal_idx >= cloud.width) 
+        {
+          continue;
```

Comment:
You should print a `PCL_WARNING` the first time you encounter a normal from a non existing point so that the user knows there are is an unexpected problem, but it's being gracefully handled.